mdoc/Makefile

```diff
@@ -960,5 +963,5 @@ check-doc-tools-update: check-monodocer-since-update \
 	check-mdoc-export-msxdoc-update \
 	check-mdoc-validate-update 
 
-check: nunit check-doc-tools
+check: nunit dotnet-test check-doc-tools
```

Comment:
some cases will be run twice (net471 and net 6.0 test dll), right?

Reply:
Yes most tests will run twice, but this is desirable as they will be running against different framework versions.